istioctl/cmd/analyze.go

```diff
@@ -315,6 +310,9 @@ func Analyze() *cobra.Command {
 		"Suppress reporting a message code on a specific resource. Values are supplied in the form "+
 			`<code>=<resource> (e.g. '--suppress "IST0102=DestinationRule primary-dr.default"'). Can be repeated. `+
 			`You can include the wildcard character '*' to support a partial match (e.g. '--suppress "IST0102=DestinationRule *.default" ).`)
+	analysisCmd.PersistentFlags().StringArrayVarP(&multiClusterContexts, "multi-cluster-contexts", "M", []string{},
```

Comment:
Is this the first istioctl command that accepts multiple clusters at once?

Reply:
Yes, this is the first one. May need to polish the flag name and the way to accept config files/contexts.